src/i18n/zh-tw/nav.ts

```diff
@@ -5,50 +5,63 @@ export default NavDictionary({
 	'getting-started': '新手上路',
 	install: '安裝',
 	'editor-setup': '編輯器設定',
+	'guides/upgrade-to/v2': '升级到 v2',
```

Comment:
This should fix the CI issue!
```suggestion
	'guides/upgrade-to/v3': '升级到 v3',
```